types/testing-library__jest-dom/index.d.ts

```diff
@@ -28,7 +28,7 @@ declare namespace jest {
         toHaveClass(...classNames: string[]): R;
         toHaveFocus(): R;
         toHaveFormValues(expectedValues: Record<string, unknown>): R;
-        toHaveStyle(css: string): R;
+        toHaveStyle(css: string | object): R;
```

Comment:
I think this might be better expressed as `string | Record<string, unknown>` similar to the one above.